business/services.go

```diff
@@ -184,6 +198,43 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 		drUpdate, drDelete = getUpdateDeletePermissions(in.k8s, namespace, DestinationRules, "")
 	}()
 
+	var eTraces int
+	go func() {
+		// Maybe a future jaeger business layer
+		defer wg.Done()
+		if config.Get().ExternalServices.Jaeger.Service != "" {
+			u, err := url.Parse(fmt.Sprintf("http://%s:%d/api/traces", config.Get().ExternalServices.Jaeger.Service, config.Get().ExternalServices.Jaeger.ServicePort))
```

Comment:
This URL is going to be http always ?
I understand that this is an internal call within the cluster (pod kiali -> jaeger service in same istio-system namespace).
But I wonder if we may want to configure http/https.

Reply:
at the moment jaeger is sevred internally with http and its the edge (or istio isntallation) who provide the https access to this internal url.